modules/300-prometheus/templates/alertmanager/internal/alertmanager.yaml

```
@@ -71,6 +71,9 @@ spec:
             alertmanager: {{ .name }}
         topologyKey: kubernetes.io/hostname
   {{- end }}
+{{- if .Values.global.modules.publicDomainTemplate }}
+  externalUrl: {{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "grafana") }}/alertmanager/{{ .name }}
+{{- end }}
```

```suggestion
{{- if $.Values.global.modules.publicDomainTemplate }}
  externalUrl: {{ include "helm_lib_module_uri_scheme" $ }}://{{ include "helm_lib_module_public_domain" (list . "grafana") }}/alertmanager/{{ .name }}
{{- end }}
```

Tests failed because of the scope of the variables. To fix this, it is possible to use the global context to access values.

---

Certainly, I didn't notice that the scope narrows down above. Thank you.